node/graphql/index.ts

```diff
@@ -74,14 +75,37 @@ export const resolvers = {
   Query: {
     getStores: async (_: any, param: any, ctx: any) => {
       const {
-        clients: { hub, sitemap },
+        clients: { hub, sitemap, vbase },
         vtex: { logger },
       } = ctx
 
+      const APP_NAME = 'store-locator'
+      const SHCEMA_NAME = 'sitemap'
+
+      const saveInVbase = () => {
+        sitemap
+          .saveIndex()
+          .then((res: any) => {
+            if (res.data.saveIndex) {
+              vbase.saveJSON(APP_NAME, SHCEMA_NAME, {
+                alreadyHasSitemap: true,
+              })
+            }
+          })
+          .catch((err) => logger.log('Error to save in vbase:', err))
+      }
```

Comment:
Although you don't care about the response from this promise, it's interesting to return it as a pattern project and keep in mind the unit tests further ahead.
Also, I added more details on the error message.


```suggestion
       const saveInVbase = () => {
          return sitemap
          .saveIndex()
          .then(async (res: any) => {
            if (res.data.saveIndex) {
              return await vbase.saveJSON(APP_NAME, SHCEMA_NAME, {
                alreadyHasSitemap: true,
              })
            }
            return false;
          })
          .catch((err) => {
             logger.log({error: err, message: 'getStores-saveInBase-error'})
             return false;
          })
      }
```

Reply:
Hello, in the line 105 i get a message: Redundant use of `await` on a return value.